### ISSUE TITLE
Add --skip-users

### ISSUE THREAD

**Issue by vsoch** [con/tributors#34]
We might want to skip adding users from a specific list, e.g., in this issue https://github.com/USRSE/usrse.github.io/pull/252 I didn't want my "robot account" added (but it was re-added anyway).

**Comment by yarikoptic** [comment]
It is hard to follow when issues are closed without a note what had closed them. Best strategy is to use `Closes #??` markers either in commit or PR messages. https://docs.github.com/en/enterprise/2.16/user/github/managing-your-work-on-github/closing-issues-using-keywords

**Comment by vsoch** [comment]
Yep I can do that. It’s hard to do here because when you open an issue I have trouble sometimes pinpointing exactly what needs to be addressed. For those I tried to guess and closed when it was done.
